docs/recommendations.md

```diff
@@ -25,63 +27,157 @@ _diffrn_measurement.integration        stepwise
+`_diffrn_measurement.tracking` is a simple yes/no code to indicate if some tracking method was used to maximize the time that the sample is illuminated by the crystal, if the crystal drifts out of the beam during data collection. Details of the method can be given as free text in the `_diffrn_measurement.tracking_method` field. _A literature reference may also be supplied with the data item `_computing.sample_tracking`._
 
-## `_diffrn_radiation.illumination_mode`
+### `_diffrn_radiation.illumination_mode`
 
 This data item is used to discriminate between convergent-beam (STEM) and parallel (TEM) illumination of the sample.
 
-## `_diffrn_radiation.probe`
+### `_diffrn_radiation.probe`
 
 This data item **should always be present with value 'electron'** for 3D ED structure determinations, to inform checkCIF of the nature of the study. The similar data item `_diffrn_radiation.type` is now deprecated and should **not** be used. More detailed description of the energy spectrum of the radiation source may be given in the data item `_diffrn_radiation_wavelength.type` if needed.
 
 
-## `_diffrn_source.description` and `_diffrn_source.make`
+### `_diffrn_source.description` and `_diffrn_source.make`
 
-These items replace the deprecated terms `_diffrn_source` and `_diffrn_source_type` respectively. 'Electron microscope' is often found in existing files as a value for `_diffrn_source` (now `_diffrn_source.description`; defined as _The general class of the source of radiation._), but 'electron gun' is technically more accurate. For `_diffrn_source.make` the usual options in electron diffraction are tungsten, LaB6, Schottky FEG (field-emission gun) or cold-effect FEG.
+These items replace the deprecated terms `_diffrn_source` and `_diffrn_source_type` respectively. 'Electron microscope' is often found in existing files as a value for `_diffrn_source` (now `_diffrn_source.description`; defined as _The general class of the source of radiation._), but 'electron gun' is technically more accurate. For `_diffrn_source.make` the usual options in electron diffraction are tungsten, LaB6, Schottky FEG (field-emission gun) or cold FEG.
 
-## `_diffrn_source.size`
+### `_diffrn_source.size`
 
 This free-text field, already present in the core dictionary, can be used to describe the appropriate measure of the beam size at the sample. For selected-aperture electron diffraction, this should be the beam width corresponding to the aperture setting, and not the total beam width.
 
-## `_diffrn_source.voltage` and `_diffrn_radiation.wavelength`
+### `_diffrn_source.voltage` and `_diffrn_radiation.wavelength`
 
 Both should be supplied for electron diffraction studies. The value of `_diffrn_radiation.wavelength` is expected to include a standard uncertainty to express the expected wavelength spread.
```

Comment:
The `_diffrn_radiation.wavelength` data name is not currently defined in the core dictionary. Alternatives:
- `_diffrn_radiation_wavelength.value` (main name)
- `_diffrn_radiation_wavelength` (alias, old name)
- `_diffrn_radiation_wavelength.wavelength` (alias, probably from mmCIF/PDBx)